lib/screens/add_exercise_screen.dart

```diff
@@ -89,38 +105,51 @@ class _AddExerciseScreenState extends State<AddExerciseScreen> {
 }
 
 class _BasicStepContent extends StatelessWidget {
-  final GlobalKey<FormState> _basicStepFormKey = GlobalKey<FormState>();
+  // final GlobalKey<FormState> _basicStepFormKey = GlobalKey<FormState>();
+  final GlobalKey<FormState> formkey;
+  _BasicStepContent({required this.formkey});
   @override
   Widget build(BuildContext context) {
+    final addExercideProvider = context.read<AddExcerciseProvider>();
+    final exerciseProvider = context.read<ExercisesProvider>();
+    final categories = exerciseProvider.categories;
+    final muscles = exerciseProvider.muscles;
     return Form(
-      key: _basicStepFormKey,
+      key: formkey,
       child: Column(
         children: [
           AddExerciseTextArea(
             onChange: (value) => print(value),
-            title: AppLocalizations.of(context).name,
+            title: '${AppLocalizations.of(context).name}*',
```

Comment:
I spoke with the guy that made the designs, we'll change a little bit on the part where the user enters the name and description. I'll write you later